src/components/PersonInfo/PersonInfo.tsx

```diff
@@ -0,0 +1,49 @@
+import classNames from 'classnames';
+import React from 'react';
+import { Person } from '../../types';
+import { PersonLink } from '../PersonLink';
+
+type Props = {
+  person: Person,
+  isPersonSelected: boolean,
+  personMother: Person | undefined,
+  personFather: Person | undefined,
+};
+
+export const PersonInfo: React.FC<Props> = ({
+  person,
+  isPersonSelected,
+  personMother,
+  personFather,
+}) => {
+  const personMotherName = person.motherName || '-';
+  const personFatherName = person.fatherName || '-';
+
+  return (
+    <tr
+      key={person.slug}
+      data-cy="person"
+      className={classNames(
+        { 'has-background-warning': isPersonSelected },
+      )}
+    >
```

Comment:
```suggestion
    <tr
      data-cy="person"
      className={classNames(
        { 'has-background-warning': isPersonSelected },
      )}
    >
```
You need to add `key` prop in `PeopleTable` component when you render the list
![image](https://user-images.githubusercontent.com/119679758/222542783-11beabb7-c678-48ec-96f1-7e8885d4a6e9.png)